src/ucp/api/ucp.h

```diff
@@ -439,6 +440,25 @@ enum {
 };
 
 
+/**
+ * @ingroup UCP_MEM
+ * @brief UCP memory mapping protection mode.
+ *
+ * The enumeration list describes the memory mapping protections supported by the @ref
+ * ucp_mem_map() function.
+ */
+enum {
+    UCP_MEM_MAP_PROT_REMOTE_WRITE = UCS_BIT(0),  /**< Enable remote write access. */
+    UCP_MEM_MAP_PROT_REMOTE_READ  = UCS_BIT(1),  /**< Enable remote read access. */
+    UCP_MEM_MAP_PROT_LOCAL_WRITE  = UCS_BIT(2),  /**< Enable local write access. */
+    UCP_MEM_MAP_PROT_LOCAL_READ   = UCS_BIT(3),  /**< Enable local read access. */
```

Comment:
define like this to allow future extension:
```
UCP_MEM_MAP_PROT_LOCAL_READ   = UCS_BIT(0)
UCP_MEM_MAP_PROT_LOCAL_WRITE  = UCS_BIT(1)

UCP_MEM_MAP_PROT_REMOTE_READ = UCS_BIT(8)
UCP_MEM_MAP_PROT_REMOTE_WRITE = UCS_BIT(9)

```

Reply:
@yosefe Fixed. Thanks!